components/bluetooth/lib.rs

```diff
@@ -366,44 +365,32 @@ impl BluetoothManager {
         None
     }
 
-    #[cfg(target_os = "linux")]
     fn select_device(&mut self, devices: Vec<BluetoothDevice>, adapter: &BluetoothAdapter) -> Option<String> {
         if is_mock_adapter(adapter) || opts::get().headless {
-            for device in devices {
+            for device in &devices {
                 if let Ok(address) = device.get_address() {
                     return Some(address);
                 }
             }
-            return None;
```

Comment:
why?

Reply:
Thanks for catching, this looks mistake I made while integrate flows of `select_device` function in bluethooth thread. Updated PR to revert changes.